main.go

```diff
@@ -73,25 +118,42 @@ func main() {
 	var stagger float64
 
 	flag.StringVar(&input, "input", "", "path to the input media")
-	flag.StringVar(&sfu, "sfu", "localhost:50051", "ion-sfu grpc url")
+	flag.StringVar(&sfu, "sfu", "50051", "ion-sfu grpc url")
 	flag.Var(&rooms, "room", "Rooms to join.")
```

Comment:
Lets remove the existing cli args and just support config.toml for now. later we can support overloading with cli flags